integration-test-groups/xml/native/classpath/src/test/java/org/apache/camel/quarkus/component/xml/it/XsltTest.java

```diff
@@ -79,10 +108,26 @@ public void xsltExtensionFunction() {
     }
 
     @Test
-    public void xsltCustomURIResolver() {
+    @DisabledOnIntegrationTest("xslt-saxon extension function does not be supported in native mode")
```

Comment:
So xslt-saxon does not support extension function in native-mode. Is there an open issue ? Or maybe something by design that can't be changed ?

Reply:
I will create an issue later.

Reply:
see https://github.com/apache/camel-quarkus/issues/5189